### PR TITLE
enhance: support httpv1/v2 throttle and add it for httpV2(#35350)

### PR DESCRIPTION
**Pull request by MrPresent-Han**
related: #35350
pr: https://github.com/milvus-io/milvus/pull/35470

### CODE REVIEW

**Comment by bigsheeper** [comment]
Maybe we can make rate limiting at wrapperProxy function, instead of limit for each request type
image

**Comment by MrPresent-Han** [comment]
> Maybe we can make rate limiting at wrapperProxy function, instead of limit for each request type image

But only insert/upsert/delete/query/search is needed for limiting, adding limiter in wrapperProxy will try to apply checkLimit for all request

**Comment by bigsheeper** [comment]
> > Maybe we can make rate limiting at wrapperProxy function, instead of limit for each request type image
> 
> But only insert/upsert/delete/query/search is needed for limiting, adding limiter in wrapperProxy will try to apply checkLimit for all request

DDL also needs (like create colleciton, load collection...)

File: internal/distributed/proxy/httpserver/handler_v2.go [hunk 1]
```diff
@@ -257,6 +261,17 @@ func wrapperProxy(ctx context.Context, c *gin.Context, req any, checkAuth bool,
 			return nil, err
 		}
 	}
+	if checkLimit {
+		_, err := CheckLimiter(ctx, req, pxy)
+		if err != nil {
+			log.Warn("high level restful api, fail to check limiter", zap.Error(err), zap.String("method", fullMethod))
```
Comment (bigsheeper) [inline internal/distributed/proxy/httpserver/handler_v2.go:267]: maybe too frequent? it's ok to remove it in next pr.

**Pull request merged**